### PR TITLE
Optimize Travis add auto release

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Optimize Travis add auto release

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/main/CONTRIBUTORS.md
[License]: ../blob/main/LICENSE

### CODE REVIEW

**Comment by mjhenkes** [comment]
4 minutes vs 9 minutes 🤷‍♂️

**Review by tbiethman — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+5. Watch main to ensure releases get pushed to npm and tagged in github appropriately
+
+If you are publishing a new package, e.g. a v1.0.0 release, the package will be listed with only you as the collaborator.
+You'll need to move the package into the terra-ui team under the Cerner org on npm.
+- Log into npm's website
+- Navigate to the terra-ui package page: https://www.npmjs.com/settings/cerner/teams/team/terra-ui/access
+- At the top of this page, there should be an input labeled, "Package". Type in the full package name and press enter.
+- This will add the package to ther terra-ui team's packages under the Cerner org on npm and then all team members on the terra-ui team in npm will have access to publish in future releases.
+
+[project-url]: https://www.npmjs.com/org/cerner/team/terra-uißß
```
Comment (benbcai) [inline RELEASE.md:20]:
```suggestion
[project-url]: https://www.npmjs.com/org/cerner/team/terra-ui
```

https://www.npmjs.com/org/cerner/team/terra-ui is broken. Should it just be https://www.npmjs.com/org/cerner/

Comment (mjhenkes) [reply RELEASE.md:20]: lol, i pulled the trigger too fast

Comment (mjhenkes) [reply RELEASE.md:20]: corrected here: https://github.com/cerner/terra-toolkit/pull/421

**Pull request merged**